route/build/src/process_ntopo.f90

```diff
@@ -212,26 +213,40 @@ SUBROUTINE augment_ntopo(nHRU,             & ! input: number of HRUs
  !write(*,'(a,1x,1PG15.7,A)') 'after reach_list: time = ', real(time1-time0,kind(dp))/real(cr), ' s'
 
  ! ---------- Compute routing parameters  --------------------------------------------------------------------
+ ! compute channel geometry parameters (width, depth, Manning's n, and floodplain slope)
+ ! (hydraulic geometry needed for all the routing methods except impulse response function)
+ do iSeg=1,nSeg
+   if (.not.meta_SEG(ixSEG%width)%varFile) &
+     structSEG(iSeg)%var(ixSEG%width)%dat(1) = wscale* sqrt(structSEG(iSeg)%var(ixSEG%totalArea)%dat(1)) ! channel width (m)
+   if (.not.meta_SEG(ixSEG%depth)%varFile) &
+     structSEG(iSeg)%var(ixSEG%depth)%dat(1) = dscale* sqrt(structSEG(iSeg)%var(ixSEG%totalArea)%dat(1)) ! channel bankfull depth (m)
+   if (.not.meta_SEG(ixSEG%man_n)%varFile) &
+     structSEG(iSeg)%var(ixSEG%man_n)%dat(1) = mann_n                                                    ! Manning's "n" paramater (unitless)
+   if (.not.meta_SEG(ixSEG%sideSlope)%varFile) &
+     structSEG(iSeg)%var(ixSEG%sideSlope)%dat(1) = 0._dp                                                 ! channel side slope h:v=slope:v [-]. now 0->rectangular channel
+   if (.not.meta_SEG(ixSEG%floodplainSlope)%varFile) &
+     structSEG(iSeg)%var(ixSEG%floodplainSlope)%dat(1) = floodplainSlope                                 ! floodplain slope
+ end do
+
+ ! replace depth with large values e.g., 10,000 [m] if floodplain mode is off
+ if (.not.floodplain) then
+   do iSeg=1,nSeg
+     structSEG(iSeg)%var(ixSEG%depth)%dat(1) = 10000._dp ! [meter]
```

Comment:
I suggest making 10000._dp it a parameter called something like large_number just to denote that the value doesn't matter. Maybe use huge?

Reply:
yes, changed this number to huge(1.0_dp)